modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -278,15 +333,42 @@ public void setAopProxy(Object aopProxy) {
 	public void undeployClientExtensionEntry(
```

Comment:
I guess is it ok to keep this as clusterable because no invocation happens through the service. Indeed, this method shall be private unless our reflection/cluster remote invocation needs it public. Annotation, along the weird logic in updateStatus() are things to remove after we merge this solution